sycl/include/sycl/ext/oneapi/matrix/matrix-intel.hpp

```diff
@@ -612,22 +612,20 @@ template <typename Group, typename T, size_t NumRows, size_t NumCols, use Use,
           layout Layout, typename T2>
 inline __SYCL_ALWAYS_INLINE void joint_matrix_fill_checked(
     Group, joint_matrix<Group, T, Use, NumRows, NumCols, Layout> &Res,
-    const T2 &Value, size_t Stride, size_t Height, size_t Width, size_t CoordX,
+    const T2 &Value, size_t Height, size_t Width, size_t CoordX,
     size_t CoordY) {
 #if defined(__SYCL_DEVICE_ONLY__)
   using storage_element_type =
       typename oneapi::detail::jm_type_interpretation_helper_trait<
           T>::storage_element_type;
   Res.spvm = __spirv_CooperativeMatrixConstructCheckedINTEL<
-      storage_element_type, T, NumRows, NumCols,
+      storage_element_type, T, CoordX, CoordY, NumRows, NumCols,
```

Comment:
```suggestion
      storage_element_type, T, NumRows, NumCols,
```
I think it shouldn't be added (these template args correspond to spirv_ops.hpp:48-51)

Reply:
Indeed, thanks!